app/models/manageiq/providers/ibm_terraform/provider.rb

```diff
@@ -15,6 +15,10 @@ class ManageIQ::Providers::IbmTerraform::Provider < ::Provider
            :url=,
            :to => :identity_endpoint, :prefix => "identity"
 
+  delegate :url,
+           :url=,
+           :to => :cpd_endpoint, :prefix => "cpd"
```

Comment:
I don't think this is right, we still need to use the "Managed Services URL" to connect to the CAM API.  This will break refresh likely

Reply:
@agrare I am not familiar with Ruby and did a blind copy based on identity url. can you suggest me on how to fix and also on how we can test the changes